src/toPercent.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Converts `fraction` to a percentage format string.
+ *
+ * @category Lang
+ * @param {number} fraction The value to process.
+ * @param {number} [minPrecision=0] - Minimum fraction digits, default is 0.
+ * @param {number} [maxPrecision=2] - Maximum fraction digits, default is 2.
+ * @returns {string} Returns the percentage string.
+ * @example
+ *
+ * toPercent(0.2)
+ * // => 20%
+ *
+ * toPercent(0.3333333)
+ * // => 33.33%
+ *
+ * toPercent(0.2, 2)
+ * // => 20.00%
+ *
+ */
+export default function toPercent(fraction: number, minPrecision: number = 0, maxPrecision: number = 2) {
```

Comment:
Note  that `minPrecision` and `maxPrecision` are not used, so this example generates an unxpected output:

```
toPercent(12, 4, 7)
// '1,200%'
```

IMO, seems like the output is very deterministic as the locals are not included, ref https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat#using_locales.